app/src/main/kotlin/com/wire/android/ui/home/conversations/MessageItem.kt

```diff
@@ -83,8 +83,12 @@ fun MessageItem(
                 }
         ) {
             Spacer(Modifier.padding(start = dimensions().spacing8x - fullAvatarOuterPadding))
+
+            val isProfileRedirectEnabled =
+                message.messageHeader.userId != null && !(message.messageHeader.isDeleted || message.messageHeader.isUnavailable)
```

Comment:
Should the clicking on the avatar be disabled when the message is deleted? For deleted message avatar is still visible so it makes the app to behave inconsistently - the user isn't deleted, only the message is, but the profile is reachable in that case 🤔

Reply:
I think isDeleted is about the user, but good point I am going to check that first

Reply:
If it turns out that it is about the user, please rename it so this confusion doesn't happen again.
If a field in a class is not about the class itself, it should have a clear name indicating what it's referring to, like:

`isSenderDeleted`

Reply:
it is about the sender, I am going to rename it to what Vitor is proposing